labgrid/remote/client.py

```diff
@@ -159,7 +159,16 @@ def print_places(self):
                 print("Place '{}':".format(name))
                 place.show(level=1)
             else:
-                print(name)
+                aliases = []
```

Comment:
remove aliases variable, we don't need to reformat from a set to a list, we can use it directly.